### PR TITLE
logger, cockpit, tests, autotools: Added Chromium/Chrome settings and bookmarks sup…

### PR DESCRIPTION
**Pull request by olivergs**
…port

### CODE REVIEW

**Comment by olivergs** [comment]
I need to do the integration tests and report back

**Comment by olivergs** [comment]
Pushed a fixing commit after integration checking and checked integration again. Now it is working.

File: logger/fleet_commander_logger.js [hunk 1]
```diff
@@ -642,6 +644,271 @@ GSettingsLogger.prototype._bus_name_disappeared_cb = function (connection, bus_n
     this.dconf_subscription_id = 0;
 }
 
+
+var ChromiumLogger = function (connmgr, datadir, namespace, interval, bookmarks_interval) {
+    this.datadir = datadir || GLib.getenv('HOME') + '/.config/chromium';
```
Comment (aruiz) [inline logger/fleet_commander_logger.js]: I'm nitpicking here, but isn't it better adding the default value in the arguments?

Comment (aruiz) [reply logger/fleet_commander_logger.js]: Ah, this is JavaScript, nevermind!

Comment (olivergs) [reply logger/fleet_commander_logger.js]: This is madness!
Madness?
This ... is ... JAVASCRIPT!

**Pull request merged**